tests/integration.rs

```diff
@@ -181,7 +181,7 @@ async fn notify_properly_sending_message() {
         .publish(
             subscribe_topic.into(),
             message,
-            4006,
```

Comment:
nit: maybe switch out to variable so it's easier to catch next time

Reply:
Will add in follow-up PR!

Reply:
https://github.com/WalletConnect/notify-server/pull/60